README.md

```diff
@@ -44,9 +45,10 @@ Create `sidebar.html` there. Inside `sidebar.html`:
         - Cars
         - Drivers
     - In `base.html` include `sidebar.html`, so all these links will be accessible on all pages.
-9. Run server, open `http://127.0.0.1:8000/`, check if the information is there and if it is correct.
-10. Run `python manage.py test` to check your code results.
+9. Check that you put empty lines at the end of each HTML file.
+10. Run server, open `http://127.0.0.1:8000/`, check if the information is there and if it is correct.
+11. Run `python manage.py test` to check your code results.
 
-NOTE: Attach screenshots of all created or modified pages to pull request. It's important to attach images not links to them. See example:
+### Note: Attach screenshots of all created or modified pages to pull request. It's important to attach images not links to them. See example:
 
```

Comment:
and somewhere - make sure that you don't push 5000+ files

Reply:
good point

Reply:
@DmytroSvirsa really I don't understand what you mean. Can you give an example of what is it about?